services/traction/api/db/models/v1/presentation_requests.py

```diff
@@ -0,0 +1,125 @@
+"""Verification requests for proof.
+
+Models of the Traction tables for Connection Invitation and related data.
+
+"""
+import uuid
+from typing import List, Optional
+from sqlalchemy.orm import selectinload
+from sqlmodel import Field, Relationship, desc
+from sqlalchemy import Column, text, String, select
+from sqlalchemy.dialects.postgresql import UUID, JSON, ARRAY
+from sqlmodel.ext.asyncio.session import AsyncSession
+
+from api.db.models.base import StatefulModel, TimestampModel
+
+from api.db.models.v1.contact import Contact
+
+from api.endpoints.models.v1.errors import (
+    NotFoundError,
+)
+
+
+class VerifierPresentationRequest(StatefulModel, TimestampModel, table=True):
+    """Verifier Presentation Request
+
+    Model for the Verifier Presentation Request table (postgresql specific
+    dialects in use).This will track verifications of proof for the Tenants.
+
+    Attributes:
+    """
+
+    __tablename__ = "verifier_presentation_request"
+
+    v_presentation_request_id: uuid.UUID = Field(
```

Comment:
There will likely be 'holder_presentation_requests' at somepoint, and presentation_request is the `correct` acapy term for the data object. 

I don't mind `verification_requests` as a 'unique' name, i'll do that.

Reply:
yes there will be a "matching" holder table, we don't need to use the full acapy name, as long as it is clear what the intention is. for instance, we are using `issuer_credential` as a table name, but it would either be `issuer_credential_offer` or `issuer_credential_exchange` if we wanted a "proper" name. so to me, `verifier_presentation` or `verifier_request` is fine, `verifier_request` would be preferred.

Reply:
`verifier_presentation` it is!